Doppler.Sap/appsettings.json

```
@@ -23,7 +23,8 @@
           "Endpoint": "BusinessPartners"
         },
         "BillingConfig": {
-          "Endpoint": "Orders"
+          "Endpoint": "Orders",
+          "NeedCreateIncomingPayments": "false"
```

You are using a string for a boolean value.

```suggestion
          "NeedCreateIncomingPayments": false
```

---

yes, my bad. I will update it.

---

Done